text/0000-standard-dependency-metadata-format.md

```diff
@@ -0,0 +1,48 @@
+# Standardized Paketo Dependency Metadata Format
+
+## Summary
+
+This proposal suggests that the Paketo project should converge on a single dependency metadata format.
+
+## Motivation
+
+There are several projects, such as the introduction of ARM or the removal of stacks, that are being discussed that would force Paketo to update the metadata for dependencies. It seems prudent that if we are going to have to do a large update to dependency metadata that it might also be a good time for us to converge as a project on the dependency metadata format that we should be using going forward.
+
+This could also be a good first step in converging on shared tooling. By having a shared dependency metadata format we would have a good common convergence point to begin building universal tooling for the Paketo project. 
+
+## Implementation
+
+The following in the proposed metadata format:
+
+```toml
+[[metadata.dependencies]]
+arch = "<dependency compatible architecture>"
+checksum = "<dependency algo:checksum>"
+id = "<dependency ID>"
+os = "<dependency compatible OS>"
+uri = "<dependency URI>"
+version = "dependency version"
+
+cpes = [ "<dependency cpe>" ] #optional
+name = "<dependency name>" #optional
```

Comment:
```suggestion
name = "<dependency name>" #optional
deprecation_date = "<dependency eol>" #optional
```

Reply:
I wonder if it makes sense to include the logic around this field?

- no deprecation_date means we don't know/can't find the date
- a depreciation_date means it ends on the given date.
- if the depreciation_date is in the past, it's EOL
- if the depreciation_date is in the future, it's not yet EOL.

Any states or implied information I'm forgetting?

Reply:
I think that logic checks out. I am going to add this as `deprecation-date` (changing to _ to - to reflect the other variables).